homework-submission/week4-homework/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./App.css";
+import TodoItem from "./components/TodoItem";
+import List from "./components/List";
+import Checkbox from "./components/Checkbox";
+import Header from "./components/Header";
+import AddTodoForm from "./components/AddTodoForm";
+import { observer, inject } from "mobx-react";
+
+@inject("TodoStore")
+@observer
+class App extends React.Component {
+  render() {
+    const { TodoStore } = this.props;
+
+    let todoItems = TodoStore.todos.map((todo, index) => (
+      <TodoItem
+        key={index}
+        {...todo}
+        onRemove={() => {
+          TodoStore.deleteTodo(todo.id);
+        }}
+        onUpdate={TodoStore.updateTodo}
+      >
+        <Checkbox
+          key={index}
+          done={todo.done}
+          onClick={() => {
+            TodoStore.toggleDone(todo.id);
```

Comment:
this line is a very nice implementing of a function call with passing in a short way instead of moving it to the side of the child component, But it's better to note about that in the comment in the child component.